README.md

```diff
@@ -62,6 +65,9 @@ Seq.of(1, 2, 3, 4).grouped(i -> i % 2);
 // (tuple(1, 1), tuple(2, 2))
 Seq.of(1, 2, 4).innerJoin(Seq.of(1, 2, 3), (a, b) -> a == b);
 
+// (tuple(1, 1), tuple(2, 2))
+Seq.of(1, 2).innerSelfJoin((t, u) -> t == u)

```

Comment:
Ill just change that to 

// (tuple(1, 2), tuple(2, 1))
Seq.of(1, 2).innerSelfJoin((t, u) -> t != u)